yarn.lock

```diff
@@ -19439,14 +19440,14 @@ __metadata:
   languageName: node
   linkType: hard
 
-"expo-local-authentication@npm:^13.3.0":
-  version: 13.4.0
-  resolution: "expo-local-authentication@npm:13.4.0"
+"expo-local-authentication@npm:^13.3.0, expo-local-authentication@npm:^13.6.0":
```

Comment:
We now have two versions of package in monorepo which could be problematic. Please try to use `yarn why` and resolve it to one version. Or maybe `yarn dedupe` will help too.

Reply:
resolved in 93d835e